lib/stamp/stamp.go

```diff
@@ -0,0 +1,22 @@
+package stamp
+
+import (
+	"strings"
+)
+
+var (
+	BuildUser    = "<unknown>"
+	GitBranch    = "<unknown>"
+	GitSha       = "<unknown>"
+	GitMasterSha = "<unknown>"
+
```

Comment:
more Im not sure you wanted a new line

Reply:
This separates public and private vars (in Go, vars that start with an uppercase are public, whereas ones that start with a lowercase letter are private to this library only)